src/arch/runtime/event_queue/iocp.hpp

```diff
@@ -1,12 +1,15 @@
 #ifndef ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 #define ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 
+#include <stdint.h>
```

Comment:
@srh Because it's not conformant to the C++ standard. Not using the correct headers creates potential incompatibilities with compilers.